### PR TITLE
Task length

### PR DESCRIPTION
**Pull request by IsraaHamdi**
RELATES#20

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -71,3 +71,10 @@ document.querySelectorAll('li').forEach((ele)=>{ele.remove()})
 
 }
 
+let lenBtn=document.querySelector('.task-num')
+lenBtn.addEventListener('click',taskNumber)
+function taskNumber(){
+  let ul =document.querySelector('ul')
```
Comment (techhana2014) [inline script.js]: you should delete this line

File: style.css [hunk 2]
```diff
@@ -257,7 +257,7 @@ header i {
   display: none;
   margin-top: 7px;
   display: none;
-  width:50px;
+  min-width:150px;
 }
```
Comment (yaqootturman) [inline style.css:265]: Using pixels not the best way to set width and sizes

**Review by yaqootturman — CHANGES_REQUESTED** [review]

File: script.js [hunk 3]
```diff
@@ -65,9 +66,14 @@ function deleteTask(index) {
 //DELETE ALL FUNCTION
 document.querySelector('.delete-all').addEventListener('click', deleteAll)
 function deleteAll(){
-document.querySelectorAll('li').forEach((ele)=>{ele.remove()})
+  list.forEach((ele)=>{ele.remove()})
```
Comment (yaqootturman) [inline script.js]: Delete comments and empty line

File: style.css (hunk 2 above)
Comment (yaqootturman) [reply style.css:265]: Dont use pixels

**Review by yaqootturman — CHANGES_REQUESTED** [review]

File: script.js (hunk 1 above)
Comment (IsraaHamdi) [reply script.js]: done

File: style.css (hunk 2 above)
Comment (IsraaHamdi) [reply style.css:265]: done

File: script.js (hunk 3 above)
Comment (IsraaHamdi) [reply script.js]: done

**Review by techhana2014 — APPROVED** [review]

**Review by Fady-Alwazir — APPROVED** [review]

**Review by yaqootturman — APPROVED** [review]

**Pull request merged**